docs/content/authorization-concepts.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Authorization Concepts
+description: Introduction to Authorization 
+sidebar_position: 1
+slug: /authorization-concepts
+---
+
+import { ProductName, ProductNameFormat, RelatedSection } from '@components/Docs';
+
+# Authorization Concepts 
+
+## Authentication vs Authorization
+
+[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or **AuthN**) ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or **AuthZ**) determines if a user can perform a certain action on a particular resource.
+
+For example, when you log in to Google, Authentication is the process of verifying that your username and password are correct. Authorization is the process of ensuring that you can access a given Google service or feature. [For more information about AuthN vs AuthZ, click here.](https://www.okta.com/identity-101/authentication-vs-authorization/)
+
+## What Is Fine-Grained Authorization?
+
+**Fine-Grained Authorization** (FGA) implies the ability of granting specific users permission to perform certain actions in specific resources.
+
+Well-designed FGA systems allow managing permissions to millions of objects and users that change rapidly as objects are added and access permissions are updated. 
+
+A notable example of fine-grained authorization is Google Drive: access can be granted either to documents or to folders, as well as to individual users or users as a group, and access rights regularly change as new documents are created and shared with specific users or groups.
```

Comment:
We go back and forth here between spelling out fine-grained authorization and using the FGA acronym. We should remain consistent, and probably just use FGA after the first declaration.